### PR TITLE
Book lock Fuzzy cat Items' recipes

### PR DESCRIPTION
**Pull request by John-Candlebury**
This proposes to booklock the recipes for the fuzzy cat items and its leather variants. Recipes for the items can be found in the Boston AnimeCon magazine, which spawns both in schools and in the general magazine itemgroup. 

Reasoning behind the change are:
-Fancy items are all uncraftable aside from these 6 cases. And these items allow you to reach +6 morale for no encumbrance penalties.
-Being generally useless save for two play styles: The aforementioned stylish min-maxing and a presumably small number of roleplayers.

### CODE REVIEW

**Comment by Soadreqm** [comment]
The magazine can only raise tailoring to 1, right? That's how much the Otaku starts with, so you might as well add it to the profession items.

File: data/json/items/book/tailor.json [hunk 1]
```diff
@@ -35,6 +35,14 @@
     "fun": 2
   },
   {
+    "id": "mag_animecon",
+    "copy-from": "mag_tailor",
+    "type": "BOOK",
+    "name": "Boston AnimeCon magazine",
+    "name_plural": "Boston AnimeCon magazines",
```
Comment (illi-kun) [inline data/json/items/book/tailor.json]: This is useless entry since the general rule to define plural name (name+"s") gives the same result.

**Comment by John-Candlebury** [comment]
mag_tailor, the item I'm copying, has that field defined, so this overwrites the original. I could copy from another definition, but mag_tailor is identical to this item so the change is likely to increase line number, and I thought we were trying our best to reduce that.

Your call.

**Comment by illi-kun** [comment]
Uh, good to know this, it's totally fine then.

**Comment by John-Candlebury** [comment]
Alright then!

Also, I noticed that my reply might come off as overly confrontative, my apology if it did, that was not my intention.

**Comment by illi-kun** [comment]
No need to worry, you correctly answered on my question. I'll finalize the review really soon.

**Pull request merged**